server/container_create.go

```diff
@@ -274,7 +274,8 @@ func (s *Server) createSandboxContainer(containerID string, containerName string
 	}
 
 	specgen.AddAnnotation("ocid/name", containerName)
-	specgen.AddAnnotation("ocid/sandbox_id", sb.id)
+	specgen.AddAnnotation("ocid/sandbox_name", sb.infraContainer.Name())
```

Comment:
Can we retain the existing ocid/sandbox_id? I have no objections to adding the new annotations.